### PR TITLE
Feat/wizzard keybindings

### PR DESCRIPTION
**Pull request by cloeser**

### CODE REVIEW

**Review by tschmidtb51 — COMMENTED** [review]
Thank you for implementing the feature.

Please have a look at my comments - as there are also context-specific keybinding (e.g. one that works in the JSON Editor tab because of the Monaco), it might be valuable to have them all in one page. See a suggested structure in the project wiki.

I haven't compiled it yet but it general it looks like we are getting to the right point.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,30 @@ the same as `"loginAvailable": false`.
...
 Please refer to [`DEVELOPMENT.md`](DEVELOPMENT.md) for a detailed description on how to build and deploy Secvisogram in production.
 
+### Configure keybindings
+
+The following hotkeys are available by default:
+
+- New / Open: STRG + O
+- Save: STRG + S
+- Export: STRG + E
+- Validate: STRG + I
```
[the remark this answers is not in the document]
Comment (cloeser) [reply README.md:86]: If we have to use two mod keys, we could also use CTRL+ALT+V?

File: README.md [hunk 2]
```diff
@@ -76,6 +76,30 @@ the same as `"loginAvailable": false`.
...
+{
+  ...,
+  "keyBindings": {
+    "keyNew": "ctrl+o",
+    "keySave": "ctrl+s"
+  }
+}
+```
+
+Possible keynames are defined in: [react-hotkeys](https://github.com/jaywcjlove/react-hotkeys#keyname)
```
[the remark this answers is not in the document]
Comment (cloeser) [reply README.md:101]: I have not found a list of all keybindings that monaco is using by default that we can copy, link or track for changes. Also there are different keybindings is used on a mac.
Instead we could instead point to the command palette that is accessible by pressing F1?

**Review by tschmidtb51 — APPROVED** [review]
From code looks good to me. 

@Reviewer 2: Please make sure (and document) it actually works when compiled. Thanks!

**Review by domachine — APPROVED** [review]

**Pull request merged**